.github/workflows/ci-trivy-container-scan.yaml

```diff
@@ -54,7 +89,7 @@ jobs:
         with:
           image-ref: ${{ env.IMAGE_REF }}
           scanners: vuln
-          severity: ${{ inputs.severity != 'ALL' && inputs.severity ||  'UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL'  }}
+          severity: ${{ inputs.severity != 'ALL' && inputs.severity ||  'ALL'  }}
```

Comment:
I don't think `ALL` is an actual severity (or at least I did not see it advertised in the [config file](https://aquasecurity.github.io/trivy/v0.50/docs/references/configuration/config-file/)).

Reply:
👍